server/src/server.js

```diff
@@ -205,6 +209,20 @@ app.get("/content/:id/:domain", function (req, res) {
   });
 });
 
+app.get("/images/:id/:domain/:clipId", function (req, res) {
+  Shot.prototype.Clip.getRawBytesForClip(
+    req.params.id, req.params.domain, req.params.clipId
+  ).then((bytes) => {
+    if (bytes === null) {
+      simpleResponse(res, "Not Found", 404);
+    } else {
+      res.header("Content-Type", "image/png");
```

Comment:
We need to store this content type, and not hard code it.  It's in the data URL.  In the future we might want to use jpeg for some clips.